Libplanet.Tests/Net/SwarmTest.cs

```diff
@@ -2757,6 +2757,105 @@ public async Task FindSpecificPeerAsync()
                 await StopAsync(swarmB);
                 await StopAsync(swarmC);
                 await StopAsync(swarmD);
+
+                swarmA.Dispose();
+                swarmB.Dispose();
+                swarmC.Dispose();
+                swarmD.Dispose();
+            }
+        }
+
+        [Fact(Timeout = Timeout)]
+        public async Task BreadcrumbTrail()
+        {
+            int swarmCount = 5;
```

Comment:
It seems better to be const variable because `swarmCount` does not change.

```suggestion
            const int swarmCount = 5;
```